app-vite/lib/modes/ssr/ssr-devserver.js

```diff
@@ -296,12 +304,10 @@ class SsrDevServer extends AppDevserver {
       })
     })
 
-    await injectMiddlewares(middlewareParams)
```

Comment:
Please remove the publicPath handling altogether.

If the middlewares run after publicPath then we kill the possibility of SSR to be able to supply other paths (eg. /api). On the other hand, if the middlewares run first, then the publicPath logic won't be able to run since the render middleware catches every route and ends the socket with html.

Reply:
On second thought, the middleware catches everything for the publicPath in use. So forget about the comment above.